build-repo.sh

```diff
@@ -86,7 +78,7 @@ then
   ); do
     echo "Resigning ${index}"
     rm -f ${index}
-    apk index -o ${index} $(find $(dirname ${index})/../ -name '*.apk')
+    apk index --allow-untrusted -o ${index} $(find $(dirname ${index})/../ -name '*.apk')
```

Comment:
Why do we need to add `--allow-untrusted` now?

Reply:
`apk index` now validates that the apks are signed, but it's not yet signed as it's resigning